zio-http/src/main/scala/zio/http/netty/NettyBodyWriter.scala

```diff
@@ -0,0 +1,72 @@
+package zio.http.netty
+
+import io.netty.buffer.Unpooled
+import io.netty.channel._
+import io.netty.handler.codec.http.LastHttpContent
+import zio._
+import zio.http.Body
+import zio.http.Body._
+
+object NettyBodyWriter {
+
+  def write(body: Body, ctx: ChannelHandlerContext): Task[Boolean] = body match {
+    case body: ByteBufBody            =>
+      ZIO.attempt(ctx.write(body.byteBufData): Unit).as(false)
+    case body: FileBody               =>
+      ZIO.attempt {
+        val file = body.fileData
+        // Write the content.
+        ctx.write(new DefaultFileRegion(file, 0, file.length()))
+
+        // Write the end marker.
+        ctx.writeAndFlush(LastHttpContent.EMPTY_LAST_CONTENT): Unit
```

Comment:
All these `: Unit` may invoke Scala's conversion of any value into unit, which will involve a function call in the best case, and an additional allocation in the worst case. I would instead just do `val _ = ...` if you just want to discard the value without a warning from the Scala compiler.